packages/core-data/src/reducer.js

```diff
@@ -436,107 +456,110 @@ let lastEditAction;
  * @return {UndoState} Updated state.
  */
 export function undo( state = UNDO_INITIAL_STATE, action ) {
+	const omitPendingRedos = ( currentState ) => {
```

Comment:
Nit: why not use named function for easier debugging?

Reply:
Function declarations in `const f = (...) =>` syntax are equivalent to named functions. I don't remember when this started, but it's been some years. See for yourself, even in Safari:

```js
(() => {
    const foo = () => 42
    const bar = foo

    function logFunctionName(fn) {
        console.log(fn.name)
    }

    logFunctionName(bar);
})() // logs 'foo'
```

(But I agree that we should name our callbacks when possible.)

Reply:
Ah cool! So all these anon functions are anon because they're not assigned to a var?